app/views/hiring_staff/sign_in/email/sessions/check_your_email.html.haml

```diff
@@ -0,0 +1,12 @@
+- content_for :page_title_prefix do
```

Comment:
`- content_for :page_title_prefix, t('translation')`

Reply:
If you like - note that lots of views do the other pattern.